xml/System.Net.Security/TlsCipherSuite.xml

```diff
@@ -18,8 +18,15 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Specifies the possible cipher suite values to negotiate with in a client-server authentication that uses the TLS/SSL protocols.</summary>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+Both, field names and values are based on the [TLS Cipher Suites list](https://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-4) posted by IANA.
```

Comment:
```suggestion
Both field names and values are based on the [TLS Cipher Suites list](https://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-4) from the Internet Assigned Numbers Authority (IANA).
```

We might also want a sentence like

```
This enumeration represents values that were known at the time a specific version of .NET was released.  
Unmapped values may be returned by properties or methods, and methods should generally not verify that a given input was defined within the enum, as the value may be transparently passed from or to the system TLS libraries.

Reply:
And maybe another one

```
Inclusion in this enumeration does not indicate that a .NET platform supports a particular cipher suite.
```

Reply:
```suggestion
Both field names and values are based on the [TLS Cipher Suites list](https://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-4) from the Internet Assigned Numbers Authority (IANA).
This enumeration represents values that were known at the time a specific version of .NET was released. Unmapped values may be returned by properties or methods, and methods should generally not verify that a given input was defined within the enum, as the value may be transparently passed from or to the system underlying implementation.
Inclusion in this enumeration does not indicate that a .NET platform supports a particular cipher suite.
```

Reply:
> We might also want a sentence like
> 
> ```
> This enumeration represents values that were known at the time a specific version of .NET was released.  
> Unmapped values may be returned by properties or methods, and methods should generally not verify that a given input was defined within the enum, as the value may be transparently passed from or to the system TLS libraries.
> ```

I would replace `system TLS libraries` for `system underlying TLS implementation`